docs/03_tutorials/tic-tac-toe-game-contract.md

```diff
@@ -709,3 +696,15 @@ $ cleos get table tic.tac.toe alice games
   "more": false
 }
 ```
+
```

Comment:
this would render in a more pleasant way instead of what we have now [here](https://github.com/EOSIO/welcome/pull/195/files#diff-c469baaaa6298561a8295631f2cb6dfbR683):
```
      "board": [
        1, 0, 0,
        0, 2, 0,
        0, 0, 0
      ]
```

Reply:
This is an output generated by cleos. Not created content

Reply:
aaaahhh ok then.
I liked how it rendered this way simulating the game table.
we could still do it but it might confuse some users which will expect the `exact` same output.